### PR TITLE
updating material-ui and react

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: desktop/renderer/index.js [hunk 1]
```diff
@@ -113,13 +115,15 @@ class Keybase extends Component {
...
 
     return (
-      <Provider store={store}>
-        <div style={{display: 'flex', flex: 1}}>
-          <RemoteManager />
-          <Nav />
-          {dt}
-        </div>
-      </Provider>
+      <MuiThemeProvider muiTheme={materialTheme}>
```
Comment (chrisnojima) [inline desktop/renderer/index.js]: material-ui now needs this wrapper for context to work like redux

File: shared/common-adapters/input.desktop.js [hunk 2]
```diff
@@ -45,6 +38,16 @@ export default class Input extends Component {
     this._textField && this._textField.blur()
   }
 
+  _onKeyDown (e: SyntheticKeyboardEvent) {
```
Comment (chrisnojima) [inline shared/common-adapters/input.desktop.js]: they deprecated onEnterKeyDown so we handle it through onkeydown

File: shared/devices/render.desktop.js [hunk 3]
```diff
@@ -200,7 +200,7 @@ const stylesPopup = {
   left: 'initial',
   right: 'initial',
   bottom: 'initial',
-  marginLeft: '-110'
```
Comment (chrisnojima) [inline shared/devices/render.desktop.js]: new react gives us a warning

**Comment by MarcoPolo** [comment]
👍

File: desktop/npm-shrinkwrap.json [hunk 4]
```diff
@@ -2307,6 +2291,11 @@
       "from": "isobject@>=2.0.0 <3.0.0",
       "resolved": "https://registry.npmjs.org/isobject/-/isobject-2.1.0.tgz"
     },
+    "isomorphic-fetch": {
```
Comment (chromakode) [inline desktop/npm-shrinkwrap.json]: I was curious about this, and found that `fbjs` now includes it. Interesting!

**Comment by chromakode** [comment]
Huh, why didn't Travis run on this pull?

**Comment by chromakode** [comment]
Ah -- it's not a pull req onto master.

**Pull request merged**